src/client/dfs/dfs.c

```diff
@@ -2431,14 +2445,16 @@ dfs_osetattr(dfs_t *dfs, dfs_obj_t *obj, struct stat *stbuf, int flags)
 	rc = daos_obj_update(oh, th, &dkey, akeys_nr, iods, sgls, NULL);
 	if (rc) {
 		D_ERROR("Failed to update attr (rc = %d)\n", rc);
-		D_GOTO(out, rc = -daos_der2errno(rc));
+		D_GOTO(err, rc = daos_der2errno(rc));
 	}
 
+out:
+
 	rc = entry_stat(dfs, th, oh, obj->name, stbuf);
 	if (rc)
-		D_GOTO(out, rc);
+		D_GOTO(err, rc);
 
-out:
+err:
```

Comment:
probably better to label those out_stat and out_obj, since it's not always err case here